CHANGELOG.md

```diff
@@ -36,7 +36,7 @@ App Center Data is [retired](https://aka.ms/MBaaS-retirement-blog-post) and has
 #### iOS
 
 * **[Improvement]** Add information logs when signals are removed/restored during initialization.
-
+* **[Feature]** Add `Crashes.UseMonoRuntimeSignalMethods` boolean property to call before `AppCenter.Start`, if set to `false` it forces the [old signal behavior](https://www.mono-project.com/docs/advanced/signals/#incomplete-solution) even if the runtime Mono version is 4.8 or higher. The default value is `true`, which means using the [current signal behavior](https://www.mono-project.com/docs/advanced/signals/#complete-solution) if the Mono version is 4.8 or higher (and falling back to the old one on older Mono versions). This is introduced in case the current signal behavior causes unexpected segmentation fault crashes.
```

Comment:
What the minimal version of mono required now? I thought it's higher than 4.8.

My thought here is it can be because of reflection. Also, we still don't know what exactly the case is, so I'm not sure that new public API is OK at this stage

Reply:
We don't really have a choice for now.

Reply:
We can always deprecate the property if we can get a root cause later, we agreed to use that workaround in the meantime in standup anyway to unblock them.

Reply:
The docs will only be in troubleshooting page also, that was the other point discussed.